src/azure-cli/azure/cli/command_modules/monitor/grammar/autoscale/AutoscaleConditionLexer.py

```diff
@@ -0,0 +1,174 @@
+        buf.write(u":\3\2\2\2\u008f\u0090\t\21\2\2\u0090<\3\2\2\2\u0091\u0092")
+        buf.write(u"\t\22\2\2\u0092>\3\2\2\2\u0093\u0094\5\65\33\2\u0094")
+        buf.write(u"\u0095\5%\23\2\u0095\u0096\5#\22\2\u0096\u0097\5/\30")
+        buf.write(u"\2\u0097\u0098\5#\22\2\u0098@\3\2\2\2\u0099\u009a\5\35")
+        buf.write(u"\17\2\u009a\u009b\5+\26\2\u009b\u009c\5!\21\2\u009cB")
+        buf.write(u"\3\2\2\2\u009d\u009e\5\'\24\2\u009e\u009f\5+\26\2\u009f")
+        buf.write(u"\u00a0\5\37\20\2\u00a0\u00a1\5)\25\2\u00a1\u00a2\5\63")
+        buf.write(u"\32\2\u00a2\u00a3\5!\21\2\u00a3\u00a4\5#\22\2\u00a4\u00a5")
+        buf.write(u"\5\61\31\2\u00a5D\3\2\2\2\u00a6\u00a7\5#\22\2\u00a7\u00a8")
+        buf.write(u"\5\67\34\2\u00a8\u00a9\5\37\20\2\u00a9\u00aa\5)\25\2")
+        buf.write(u"\u00aa\u00ab\5\63\32\2\u00ab\u00ac\5!\21\2\u00ac\u00ad")
+        buf.write(u"\5#\22\2\u00ad\u00ae\5\61\31\2\u00aeF\3\2\2\2\u00af\u00b0")
+        buf.write(u"\5-\27\2\u00b0\u00b1\5/\30\2\u00b1H\3\2\2\2\u00b2\u00bd")
+        buf.write(u"\7>\2\2\u00b3\u00b4\7>\2\2\u00b4\u00bd\7?\2\2\u00b5\u00b6")
+        buf.write(u"\7?\2\2\u00b6\u00bd\7?\2\2\u00b7\u00b8\7@\2\2\u00b8\u00bd")
+        buf.write(u"\7?\2\2\u00b9\u00bd\7@\2\2\u00ba\u00bb\7#\2\2\u00bb\u00bd")
+        buf.write(u"\7?\2\2\u00bc\u00b2\3\2\2\2\u00bc\u00b3\3\2\2\2\u00bc")
+        buf.write(u"\u00b5\3\2\2\2\u00bc\u00b7\3\2\2\2\u00bc\u00b9\3\2\2")
+        buf.write(u"\2\u00bc\u00ba\3\2\2\2\u00bdJ\3\2\2\2\u00be\u00c0\59")
+        buf.write(u"\35\2\u00bf\u00be\3\2\2\2\u00c0\u00c1\3\2\2\2\u00c1\u00bf")
+        buf.write(u"\3\2\2\2\u00c1\u00c2\3\2\2\2\u00c2\u00c9\3\2\2\2\u00c3")
+        buf.write(u"\u00c5\t\23\2\2\u00c4\u00c6\59\35\2\u00c5\u00c4\3\2\2")
+        buf.write(u"\2\u00c6\u00c7\3\2\2\2\u00c7\u00c5\3\2\2\2\u00c7\u00c8")
+        buf.write(u"\3\2\2\2\u00c8\u00ca\3\2\2\2\u00c9\u00c3\3\2\2\2\u00c9")
+        buf.write(u"\u00ca\3\2\2\2\u00caL\3\2\2\2\u00cb\u00cc\t\24\2\2\u00cc")
+        buf.write(u"N\3\2\2\2\u00cd\u00cf\t\25\2\2\u00ce\u00cd\3\2\2\2\u00cf")
+        buf.write(u"\u00d0\3\2\2\2\u00d0\u00ce\3\2\2\2\u00d0\u00d1\3\2\2")
+        buf.write(u"\2\u00d1P\3\2\2\2\u00d2\u00d4\7\17\2\2\u00d3\u00d2\3")
+        buf.write(u"\2\2\2\u00d3\u00d4\3\2\2\2\u00d4\u00d5\3\2\2\2\u00d5")
+        buf.write(u"\u00d8\7\f\2\2\u00d6\u00d8\7\17\2\2\u00d7\u00d3\3\2\2")
+        buf.write(u"\2\u00d7\u00d6\3\2\2\2\u00d8\u00d9\3\2\2\2\u00d9\u00d7")
+        buf.write(u"\3\2\2\2\u00d9\u00da\3\2\2\2\u00daR\3\2\2\2\u00db\u00e0")
+        buf.write(u"\5;\36\2\u00dc\u00e0\5=\37\2\u00dd\u00e0\59\35\2\u00de")
+        buf.write(u"\u00e0\7a\2\2\u00df\u00db\3\2\2\2\u00df\u00dc\3\2\2\2")
+        buf.write(u"\u00df\u00dd\3\2\2\2\u00df\u00de\3\2\2\2\u00e0\u00e1")
+        buf.write(u"\3\2\2\2\u00e1\u00df\3\2\2\2\u00e1\u00e2\3\2\2\2\u00e2")
+        buf.write(u"T\3\2\2\2\r\2\u00bc\u00c1\u00c7\u00c9\u00d0\u00d3\u00d7")
+        buf.write(u"\u00d9\u00df\u00e1\2")
```

Comment:
Magic

Reply:
What is it?

Reply:
This is auto-generated by [Antrl4](https://www.antlr.org/) tool.